NineChronicles.Headless.Executable/appsettings.json

```diff
@@ -87,8 +92,11 @@
         "Confirmations": 0,
         "ChainTipStaleBehaviorType": "reboot"
     },
+    "RpcRateLimiting": {
+        "EnableRateLimiting": false
+    },
     "IpRateLimiting": {
-        "EnableEndpointRateLimiting": true,
+        "EnableRateLimiting": false,
```

Comment:
Is it okay to modify too?

https://github.com/stefanprodan/AspNetCoreRateLimit/blob/dc0d24253c8d4762359dae82b559e961a270cd66/src/AspNetCoreRateLimit/Models/RateLimitOptions.cs#L58

Reply:
https://github.com/planetarium/9c-infra/blob/main/9c-main/chart/templates/remote-headless.yaml#L110
For some reason, it's been working fine with the wrong configuration... 😅 I'll adjust the values to match the package. Thanks!